### ISSUE TITLE
Copying/Cloning Characters

### ISSUE THREAD

**Issue by christhekeele** [Mottokrosh/SheetIssues#17]
It'd be nice to be able to copy a character, or import a downloaded `.json` as a new character, to experiment with other peoples' builds or plan a new level of an existing character.

**Comment by leo60228** [comment]
+1

**Comment by sturlabragason** [comment]
+1

**Comment by emarsk** [comment]
Absolutely. Being able to export a JSON is pretty useless without an import option.
